pkg/gen/input/makefile/internal/file/Makefile.gen.go.mk.template

```diff
@@ -104,8 +104,8 @@ $(PACKAGE_DIR)/$(APPLICATION)-v$(VERSION)-windows-amd64.zip: $(APPLICATION)-v$(V
 		  quay.io/giantswarm/signcode-util:latest \
 		  sign \
 		  -pkcs12 /mnt/certs/code-signing.p12 \
-		  -n "Giant Swarm plugin for kubectl" \
-		  -i https://github.com/giantswarm/kubectl-gs \
```

Comment:
We may have other windows binaries at some point so this should be generic. Completely missed it the first time.